common/OpTestSystem.py

```diff
@@ -1394,7 +1394,10 @@ def __init__(self,
                  host=None,
                  bmc=None,
                  conf=None,
+                 bmc_type=None,
                  state=OpSystemState.UNKNOWN):
+        if bmc_type in ['FSP_PHYP']:
```

Comment:
Does this require a check for EBMC_PHYP as well ?

Reply:
no @sacsant as this path holds good on `FSP_PHYP` case and earlier use case was FSP_PHYP but now same function used by ebmc as well so this condition require

Reply:
@PraveenPenguin the user will get confused,  as the system is type BMC with PHYP ..than it should be EBMC_PHYP instead FSP_PHYP ?